lib/logstash/outputs/elasticsearch.rb

```diff
@@ -165,43 +122,8 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
 
   config :host, :validate => :array
```

Comment:
@andrewvc what are your thoughts on deprecating `host` and introducing `hosts`. See https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/172